frontend/talentsearch/src/js/components/applicantProfile/CancelButton.tsx

```diff
@@ -16,14 +16,15 @@ const CancelButton = ({ href, children }: CancelButtonProps) => {
     <Link
       href={href || profilePaths.myProfile()}
       color="secondary"
-      mode="outline"
-      data-h2-display="s(inline-flex)"
-      data-h2-width="b(auto)"
-      data-h2-align-items="b(center)"
+      mode="inline"
+      data-h2-display="p-tablet(inline-flex)"
+      data-h2-width="base(auto)"
+      data-h2-align-items="base(center)"
+      data-h2-padding="base(0)"
       type="button"
     >
       <ArrowCircleLeftIcon style={{ width: "1rem" }} />
-      <span data-h2-margin="b(left, xxs)">
+      <span data-h2-margin="base(auto, auto, auto, x.125)">
         {children ||
```

Comment:
this change appears to have caused Cancel to lose its button appearance
I think it might have been the change to the value passed in for `mode`

![image](https://user-images.githubusercontent.com/40485260/185456144-7c1abbdf-e630-44f2-8171-81b64c45b052.png)

Reply:
Nope - this was a deliberate change - the cancel buttons were demanding too much focus as full buttons, so we reduced their footprint by making them inline. Good catch though!